### PR TITLE
add work and about section

### PR DESCRIPTION
**Pull request by kingsleyocran**
- Add about section to index.html
- Add work section to index.html
- Modified style.css with design and layout requirements of about and work section.
- Add new web assets; images and svgs.

### CODE REVIEW

File: css/build/style.css [hunk 1]
```diff
@@ -199,8 +263,148 @@ p {
...
+  border-radius: 8px;
+  padding: 12px;
+  margin: 12px 0;
+  transition-duration: 0.4s;
+}
+
+.work-card-button:hover {
+  background-color: #6070ff;
+  color: white;
+}
```
Comment (vechicin) [inline css/build/style.css:335]: Great job implementing the hover feature, really smooth transition, looks great! 🎉 
However, you're still missing the `active` interaction. Please add it.

Screen Shot 2022-05-26 at 14 39 29

File: index.html [hunk 2]
```diff
@@ -134,6 +137,329 @@ <h1 class="banner-header banner-content">
...
+        <h1 class="banner-header">About Myself</h1>
+
+        <p class="body-text-2">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <p class="head-text-1">LET'S CONNECT</p>
```
Comment (vechicin) [inline index.html]: According to the review guidelines, this text should be wrapped inside a `<hx>` tag. Kindly fix this.

Screen Shot 2022-05-26 at 14 42 30

File: index.html [hunk 3]
```diff
@@ -134,6 +137,329 @@ <h1 class="banner-header banner-content">
...
+                  class="banner-social-icon"
+                  src="assets/icons/discord.svg"
+                  alt="Github"
+              /></a>
+            </li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="button" class="work-card-button button-text-1">See Project</button>
```
Comment (vechicin) [inline index.html]: The comment for the button above applies here as well. Additionally, the text in the template is `Get my resume`, not `See Project`. Kindly fix this.

File: index.html [hunk 4]
```diff
@@ -134,6 +137,329 @@ <h1 class="banner-header banner-content">
...
+              <div class="about-collapsible-tools">
+                <img src="assets/images/tool-html.png" alt="javaScript" />
+                <p class="sub-text-1">HTML</p>
+              </div>
+
+              <div class="about-collapsible-tools">
+                <img src="assets/images/tool-css.png" alt="javaScript" />
+                <p class="sub-text-1">CSS</p>
+              </div>
+            </div>
```
Comment (vechicin) [inline index.html]: According to the evaluation guidelines, this section should be wrapped inside `<ul>` and `<li>` tags as well.

Screen Shot 2022-05-26 at 14 46 29

Kindly fix this.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @kingsleyocran  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Correct GitHub Flow
- [x] Fantastic job following the design template
- [x] Good PR title/description
- [x] Good README
- [x] Highly organized codebase 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/kingsleyocran/mc_portfolio/blob/029605fbed7448458b643e5784fa2e9aaa27c037/README.md?plain=1#L16-L18

I recommend you remove this section until you deploy an actual demo.

**Comment by vechicin** [comment]
https://github.com/kingsleyocran/mc_portfolio/blob/029605fbed7448458b643e5784fa2e9aaa27c037/README.md?plain=1#L4-L6

Please don't forget to add a project name and description.

**Comment by vechicin** [comment]
https://github.com/kingsleyocran/mc_portfolio/blob/029605fbed7448458b643e5784fa2e9aaa27c037/README.md?plain=1#L46-L50

I suggest you remove these unused headers, it'll keep your documentation clean which is desirable.

**Comment by vechicin** [comment]
https://github.com/kingsleyocran/mc_portfolio/blob/029605fbed7448458b643e5784fa2e9aaa27c037/README.md?plain=1#L54-L59

Please change `Author` to your full name. It'll give you more visibility. Add your social media handles as opposed to the generic `@githubhandle` or `LinkedIn` tags, it'll make your documentation look more professional.

**Review by OLIPLICHE — APPROVED** [review]
Hi @kingsleyocran , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**